torch/csrc/jit/passes/dead_code_elimination.cpp

```diff
@@ -28,6 +29,12 @@ class DeadCodeEliminator {
     sweep(block, recurse);
```

Comment:
(Leaving this here because github won't let me comment where I want to)

what is a "last wildcard"?

Reply:
(Don't ask me)

Reply:
Alias analysis has a concept of a wildcard, which is an alias set that we don't have enough information about so we assume it may alias anything. Today the most common source of wildcards is when we extract a value from a list.

Once we witness a wildcard node it's not safe to DCE any node that mutably writes, since it may write to the wildcard.

Reply:
got it, thanks!